pkg/sql/regionliveness/prober.go

```diff
@@ -38,6 +38,12 @@ var RegionLivenessEnabled = settings.RegisterBoolSetting(settings.ApplicationLev
 	false, /* disabled */
 	settings.WithVisibility(settings.Reserved))
 
+var RegionLivenessProbeTimeout = settings.RegisterDurationSetting(settings.ApplicationLevel,
+	"sql.region_liveness.probe.timeout",
+	"enables region liveness for system databases (experimental)",
```

Comment:
the description is incorrect - this is a timeout, not an enable/disable. also, i think we can remove the word experimental from this description, since this is a hidden setting anyway